hedera-node/src/main/java/com/hedera/services/context/init/StateInitializationFlow.java

```diff
@@ -54,8 +56,11 @@ public StateInitializationFlow(
         this.fileUpdateInterceptors = fileUpdateInterceptors;
     }
 
-    public void runWith(ServicesState activeState) {
-        staticNumbersHolder.accept(hederaNums);
+    public void runWith(
+            final ServicesState activeState, final BootstrapProperties bootstrapProperties) {
+        final var lastThrottleExempt =
```

Comment:
Might be better to add comment here for using `ACCOUNTS_LAST_THROTTLE_EXEMPT = 100`